client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -26,11 +26,36 @@ const INSTALL_STATE_COMPLETE = 1;
 const INSTALL_STATE_INCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
-	componentDidUpdate( prevProps ) {
-		if ( prevProps.progressComplete < 100 && this.props.progressComplete >= 100 ) {
-			this.props.recordTracksEvent( 'calypso_plans_autoconfig_success', {
+	tracksEventSent = false;
+
+	recordAutoconfigTracksEvent( eventName, options = {} ) {
+		if ( ! this.tracksEventSent ) {
+			this.tracksEventSent = true;
+			this.props.recordTracksEvent( eventName, {
 				checklist_name: 'jetpack',
 				location: 'JetpackChecklist',
+				...options,
+			} );
+		}
+	}
+
+	componentDidUpdate( prevProps ) {
+		const { progressComplete, site } = this.props;
+
+		if ( prevProps.progressComplete < 100 && progressComplete >= 100 ) {
+			this.recordAutoconfigTracksEvent( 'calypso_plans_autoconfig_success' );
+		} else if ( site && ! site.hasMinimumJetpackVersion ) {
+			this.recordAutoconfigTracksEvent( 'calypso_plans_autoconfig_error', {
+				error: 'jetpack_version_too_old',
+				jetpack_version: get( site, [ 'options', 'jetpack_version' ] ),
```

Comment:
I'm not sure, but we might get the value `"undefined"` in tracks. Maybe default to `"[unknown version]"` or something?

Reply:
Good idea, added `unknown`